clusters/pks-gcp/configure.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+SYSTEM_INSTALL_FLAGS="${SYSTEM_INSTALL_FLAGS:-}"
+
+wait_for_ingress_ready() {
+  name=$1
+  namespace=$2
+
+  wait_for_service_ip $name $namespace
+}
+
+# Install pks cli
+source `dirname "${BASH_SOURCE[0]}"`/../../install.sh pks
```

Comment:
same, what about `./`?

Reply:
The tool will be installed with a sub shell, we can avoid the extra sub shell by sourcing.